chef/data_bags/crowbar/template-ntp.json

```diff
@@ -3,14 +3,15 @@
   "description": "Common NTP service for the cluster. An NTP server or servers can be specified and all other nodes will be clients of them.",
   "attributes": {
     "ntp": {
-      "external_servers": [ ]
+      "external_servers": [ ],
+      "listen_networks": [ ]
```

Comment:
Can we actually put `"admin"` as default value there? This would make this completely configurable, so the user could drop the admin network from there too.

This would require a validation check that `listen_networks` is not empty.

Reply:
Right now this only applies to the ntp server node and the client nodes still listen only on the admin interface to synchronize with the admin node (or whatever nodes have the ntp-server role). You think that the clients should also be able to listen on a different network?

Reply:
Ah I missed that, sorry. Please discard that part of my comment :-)

Reply:
That being said, it may be good to rename the attribute to clarify that it's only for reaching the external servers?

Reply:
++